ext-src/NexusExplorer.ts

```diff
@@ -142,8 +142,8 @@ export class NexusExplorer {
     const _channel = vscode.window.createOutputChannel(`Sonatype IQ Extension`);
     context.subscriptions.push(_channel);
 
-    this.logger = new Logger({outputChannel: _channel});
-    this.logger.setLogLevel(LogLevel.ERROR);
+    this.logger = new Logger({outputChannel: _channel, logFilePath: context.storagePath!});
+    this.logger.setLogLevel(LogLevel.TRACE);
```

Comment:
I have this set to something very verbose ATM, the last step in getting logging done will be to add a setting in configuration where someone can control it.

Reply:
Configurable in: ffcdef4